lib/plugin/autoDelay.js

```diff
@@ -4,14 +4,7 @@ const recorder = require('../recorder');
 const event = require('../event');
 const log = require('../output').log;
 
-const supportedHelpers = [
-  'WebDriver',
-  'WebDriverIO',
-  'Protractor',
-  'Appium',
-  'Nightmare',
-  'Puppeteer',
-];
+const supportedHelpers = require('./supportedHelpersList');
```

Comment:
can we specify somehow supported for what? 
in OOP I would use interfaces to describe what kind of behavior is supported but here I have no ideas.

We can name helpers like: `standardActingHelpers` so we assume they have standard actions like `click`, `fillField`, `saveScreenshot`, etc

Reply:
@DavertMik I updated the name as request.